src/ServiceControl.Transports.AzureServiceBusStandard/ServiceControl.Transports.AzureServiceBusStandard.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NServiceBus.Transport.AzureServiceBus" Version="1.0.0-alpha0164" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Nitpick, but I prefer to see project references first, then package references.

Reply:
done

Reply:
done :)